File: Folder Java/Film.java
// deklarasi class
public class Film
{
    // private atribut
    private String idFilm;
    private String judul;
    private String genre;
    private int durasi;
    private String sutradara;

    // constructor
    public Film(String idFilm, String judul, String genre,
                int durasi, String sutradara)
    {
        this.idFilm = idFilm; // inisialisasi
        this.judul = judul; // inisialisasi
        this.genre = genre; // inisialisasi
        this.durasi = durasi; // inisialisasi
        this.sutradara = sutradara; // inisialisasi
    }

    // getter
    public String getIdFilm()
    {
        return idFilm; // mengambil value
    }

    public String getJudul()
    {
        return judul; // mengambil value
    }

    public String getGenre()
    {
        return genre; // mengambil value
    }

    public int getDurasi()
    {
        return durasi; // mengambil value
    }

    public String getSutradara()
    {
        return sutradara; // mengambil value
    }

    // prosedur untuk mengupdate data
    public void update(String judulBaru, String genreBaru,
                       int durasiBaru, String sutradaraBaru)
    {
        judul = judulBaru;
        genre = genreBaru;
        durasi = durasiBaru;
        sutradara = sutradaraBaru;
    }

    // prosedur untuk menampilkan data
    public void tampilkanData()
    {
        System.out.println("ID Film    : " + getIdFilm());
        System.out.println("Judul      : " + getJudul());
        System.out.println("Genre      : " + getGenre());
        System.out.println("Durasi     : " + getDurasi() + " menit");
        System.out.println("Sutradara  : " + getSutradara());
    }
}

File: Folder CPP/Film.cpp
#include <iostream>
#include <string>

using namespace std;

// deklarasi class
class Film
{
    private:
        // private atribut
        string id_film;
        string judul;
        string genre;
        int durasi;
        string sutradara;

    public:
        // constructor
        Film(string id, string judul, string genre, int durasi,
             string sutradara)
        {
            this->id_film = id; // inisialisasi
            this->judul = judul; // inisialisasi
            this->genre = genre; // inisialisasi
            this->durasi = durasi; // inisialisasi
            this->sutradara = sutradara; // inisialisasi
        }

        // getter
        string getId() const
        {
            return id_film; // mengambil value
        }

        string getJudul() const
        {
            return judul; // mengambil value
        }

        string getGenre() const
        {
            return genre; // mengambil value
        }

        int getDurasi() const
        {
            return durasi; // mengambil value
        }

        string getSutradara() const
        {
            return sutradara; // mengambil value
        }

        // prosedur untuk mengupdate data
        void update(string judulBaru, string genreBaru,
                    int durasiBaru, string sutradaraBaru)
        {
            judul = judulBaru; // mengubah judul
            genre = genreBaru; // mengubah genre
            durasi = durasiBaru; // mengubah durasi
            sutradara = sutradaraBaru; // mengubah sutradara
        }

        // prosedur untuk menampilkan data
        void tampilkanData() const
        {
            // print
            cout << "ID Film    : " << getId() << endl
                 << "Judul      : " << getJudul() << endl
                 << "Genre      : " << getGenre() << endl
                 << "Durasi     : " << getDurasi() << " menit" << endl
                 << "Sutradara  : " << getSutradara() << endl;
        }
};